app/views/devise/registrations/_registration_form.html.erb

```diff
@@ -22,23 +30,50 @@
       <%= render partial: "shared/authentication/providers_registration_form" %>
 
       <% if params[:state] == "new-user" %>
-        <div class="registration__hr">
-          <span class="registration__hr-label">
-            Already have an account? <a href="<%= sign_up_path %>">Log in</a>.
-          </span>
+
+        <div class="mt-6 align-center fs-s fw-normal fs-italic px-0 s:px-9 color-secondary">
+          By signing up, you are agreeing to our <a href="<%= privacy_path %>">privacy policy</a>, <a href="<%= terms_path %>">terms of use</a> and <a href="<%= code_of_conduct_path %>">code of conduct</a>.
+        </div>
+
+        <hr class="registration__hr" />
+
+        <div class="crayons-subtitle-3 color-grey-700 fw-normal align-center">
+          Already have an account? <a href="<%= sign_up_path %>">Log in</a>.
         </div>
+
       <% else %>
         <div class="registration__actions-email" id="sign-in-password-form">
           <% if Settings::Authentication.allow_email_password_login %>
-            <div class="registration__hr">
-              <span class="registration__hr-label">
-                Have a password? Continue with your email address
-              </span>
+            <div class="registration__hr-container">
+              <hr class="registration__hr" />
+              <div class="registration__hr-label">OR</div>
             </div>
             <%= render partial: "shared/authentication/email_login_form" %>
+
+            <div class="mt-6 align-center fs-s fw-normal fs-italic px-0 s:px-9 color-secondary">
+              By signing in, you are agreeing to our <a href="<%= privacy_path %>">privacy policy</a>, <a href="<%= terms_path %>">terms of use</a> and <a href="<%= code_of_conduct_path %>">code of conduct</a>.
+            </div>
+
+            <hr class="registration__hr" />
+
+            <div class="rcrayons-subtitle-3 color-grey-700 fw-normal align-center">
+              New to <%= community_name %>? <a href="/enter?state=new-user">Create account</a>.
+            </div>
           <% end %>
         </div>
       <% end %>
     </div>
   </div>
 </section>
+
+<script>
+  const registrationsElement = document.querySelector('.registrations');
```

Comment:
(nitpick, non-blocking):  Something that we've discussed in the past but I'm not sure that we've ever documented is the use of js- prepended classes for classes that we use in JavaScript. When looking at the HTML markup it makes it more obvious to know which classes our JS depends on.

Open to having an engineering roundtable discussion about this if folks don't feel similarly.



Also I'm wondering should we have a `javascript_packs_with_chunks_tag` for the JS in this file instead of inlining it?

Reply:
I have updated the code to use `js-registration` instead of `registrations`.